packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -44,7 +81,13 @@ contract SystemConfig is OwnableUpgradeable, Semver {
     }
 
     /**
-     * @notice Initializer;
+     * @notice Initializer.
+     *
+     * @param _owner       Initial owner of the contract.
+     * @param _overhead    Initial overhead value.
+     * @param _scalar      Initial scalar value.
+     * @param _batcherHash Initial batcher hash.
+     * @param _gasLimit    Initial gas limit.
      */
     function initialize(
```

Comment:
@smartcontracts What do you think about calling the methods instead of setting the values directly?

Reply:
I think that makes sense but let's not change that here